dtmi/stmicroelectronics/steval_proteus1/standard_fw-1.json

```diff
@@ -0,0 +1,15 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:stmicroelectronics:steval_proteus1:standard_fw;1",
+  "@type": "Interface",
+  "displayName": "STEVAL-PROTEUS1",
+  "description": "2.4GHz industrial sensor node for condition monitoring.",
+  "comment": "from proteus-poc iotc draftv2 (no deviceinfo component)",
+  "contents": [
+    {
+      "@type": "Component",
+      "name": "stdcomp",
```

Comment:
camelCase is preferred for "name" values, so perhaps this should be "stdComp" ?

Reply:
changed to std_comp, according to our current software conventions

Reply:
Fine by me.